lib/FusionInventory/Agent/Tools.pm

```diff
@@ -111,18 +111,18 @@ sub getCanonicalManufacturer {
     if (exists $manufacturers{$manufacturer}) {
         $manufacturer = $manufacturers{$manufacturer};
     } elsif ($manufacturer =~ /(
-        maxtor    |
-        sony      |
-        compaq    |
-        ibm       |
-        toshiba   |
-        fujitsu   |
-        lg        |
-        samsung   |
-        nec       |
-        transcend |
-        matshita  |
-        hitachi   |
+        maxtor     |
+        sony       |
+        compaq     |
+        ibm        |
+        toshiba    |
+        fujitsu    |
+        lg(?:\s|$) |
```

Comment:
Can you test if `lg\b` could be better here ? `\b`is the regex special escape to match word boundary